docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -0,0 +1,115 @@
+.. _lora_finetune_recipe_label:
+
+=============================
+LoRA Single Device finetuning
+=============================
+
+This recipe supports finetuning on next-token prediction tasks using `LoRA <https://arxiv.org/abs/2106.09685>`_,
+a technique to significantly reduce memory consumption during training whilst still maintaining competitive performance.
+
+Interested in using this recipe? Check out some of our awesome tutorials to show off how it can be used:
+
+* :ref:`Finetuning Llama2 with LoRA<lora_finetune_label>`
+* :ref:`End-to-End Workflow with torchtune<dataset_tutorial_label>`
+* :ref:`Fine-tuning Llama3 with Chat Data<chat_tutorial_label>`
+* :ref:`Meta Llama3 in torchtune<llama3_label>`
+* :ref:`Fine-Tune Your First LLM<finetune_llama_label>`
+
+We provide the following configurations out-of-the box. To learn how to customize recipes to suit your needs, check out
+our :ref:`configs tutorial <config_tutorial_label>`:
+
+.. code-block:: bash
+
+    lora_finetune_single_device          llama2/7B_lora_single_device
```

Comment:
I don't like how this looks here. I'm almost fine with us not including it at all and just pointing to the tune CLI like we're already doing. If we do include it, it should be small. Maybe a table of download links?